### PR TITLE
Updated php to 7.1.26 and 7.2.14

### PR DESCRIPTION
**Pull request by vaighir**

### CODE REVIEW

File: docker/shared_steps/install_php_extensions.sh [hunk 1]
```diff
@@ -28,6 +28,8 @@ php_install_extensions() {
   local already_installed_extensions=`php -m | egrep '^([A-Za-z_]+)$' | tr '[:upper:]' '[:lower:]'`
   local php_extensions_count=`echo $extensions | wc -w`
   local php_extensions_counter="1"
+  
+  mkdir '/usr/local/etc/php/conf.d/conf.d'
```
Comment (chriscoester) [inline docker/shared_steps/install_php_extensions.sh]: I think we should move this to a separate function for the relevant extension.
Such as `php_install_ldap()`. This way we know, which extension needs the fix and keep the main function clean.
Please use the `-p` flag aswell.

Comment (tonyfahrion) [reply docker/shared_steps/install_php_extensions.sh]: @vaighir why is this necessary with the new PHP versions? Or was it a bug before?

Comment (vaighir) [reply docker/shared_steps/install_php_extensions.sh]: I'm not sure why. In PHP 7.2.11 it's not necessary, but from 7.2.12 up the build fails with: 
`/usr/local/bin/docker-php-ext-enable: 108: /usr/local/bin/docker-php-ext-enable: cannot create /usr/local/etc/php/conf.d/conf.d/docker-php-ext-bcmath.ini: Directory nonexistent
BUILD FAILED!!!`

Comment (tonyfahrion) [reply docker/shared_steps/install_php_extensions.sh]: would be interesting, what change caused this - as it looks like a typo `conf.d/conf.d`

**Comment by tonyfahrion** [comment]
very good! lgtm too 🙂

**Comment by tonyfahrion** [comment]
@vaighir please fill out the `CHANGELOG.md` and bump the version (after that) to `1.1.15`. Please checkout https://github.com/claranet/php/blob/master/CONTRIBUTING.md

**Comment by tonyfahrion** [comment]
very good, will merge now

**Pull request merged**